moveit_planners/sbpl/ros/sbpl_interface_ros/src/sbpl_meta_plugin.cpp

```diff
@@ -50,8 +50,7 @@ class SBPLMetaPlanner : public planning_interface::Planner
   void init(const planning_models::RobotModelConstPtr& model)
   {
     ros::NodeHandle nh;
-    // display_bfs_publisher_ = nh.advertise<visualization_msgs::Marker>("planning_components_visualization", 10, true);
```

Comment:
Why would you remove that line? Same as with the one in OMPL we should cleanup the context for this in a different PR if we remove commented-out code.